worker/src/test/java/org/apache/celeborn/service/deploy/worker/storage/memory/MemoryReducePartitionDataWriterSuiteJ.java

```diff
@@ -526,6 +526,9 @@ public void testEvictAndChunkRead() throws Exception {
     result.releaseBuffers();
 
     closeChunkServer();
+
+    assert storageManager.evictedFileCount().get() > 0;
+    assert MemoryManager.instance().getMemoryFileStorageCounter() == 0;
```

Comment:
The UT can not pass before this patch.
<img width="1414" alt="image" src="https://github.com/user-attachments/assets/fb119026-3b99-4c99-b771-eab85f2723d6">